### PR TITLE
chore: fix .gitignores

### PR DESCRIPTION
**Pull request by eseidel**
I forgot the one for rust and fixed the one for Dart per Felix's suggestion.

### CODE REVIEW

File: updater/dart_cli/.gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # https://dart.dev/guides/libraries/private-files
 # Created by `dart pub`
 .dart_tool/
+pubspec.lock
```
Comment (felangel) [inline updater/dart_cli/.gitignore:4]: Minor but can we also `git rm --cached updater/dart_cli/pubspec.lock`?

Comment (eseidel) [reply updater/dart_cli/.gitignore:4]: Yes, I thought I had.  🤔

Comment (felangel) [reply updater/dart_cli/.gitignore:4]: weird...maybe you forgot to commit the changes? I would've expected to see the deleted lockfile as part of this diff 🤔

Comment (eseidel) [reply updater/dart_cli/.gitignore:4]: Donno.  Fixed.

**Review by felangel — APPROVED** [review]

**Pull request merged**